config/monica.php

```diff
@@ -285,4 +285,14 @@
     */
     'adorable_api' => env('ADORABLE_API', 'https://api.hello-avatar.com/adorables/'),
 
+    /*
+    |--------------------------------------------------------------------------
+    | Export limit
+    |--------------------------------------------------------------------------
+    |
+    | Maximum number of exports.
```

Comment:
This is not clear. Max number of exports... total? at the same time?

Also, you don't provide an env variable for this setting. Is this on purpose?

Reply:
I've updated the comment